src/ReactWrapper.jsx

```diff
@@ -531,13 +589,8 @@ class ReactWrapper {
    */
   simulate(event, mock = {}) {
     this.single('simulate', (n) => {
-      const mappedEvent = mapNativeEventNames(event);
-      const eventFn = Simulate[mappedEvent];
-      if (!eventFn) {
-        throw new TypeError(`ReactWrapper::simulate() event '${event}' does not exist`);
-      }
-
-      eventFn(findDOMNode(n), mock);
+      this.renderer.simulateEvent(n, event, mock);
+      this.root.update();
```

Comment:
nothing really is guaranteed other than `simulatedEvent()` finishing here... which will call the event I believe (at least our tests indicate), but of course the event could schedule something async